aten/src/ATen/Parallel.cpp

```diff
@@ -60,4 +60,29 @@ size_t get_num_threads() {
 #endif
 }
 
+PTThreadPool::PTThreadPool(
+    std::size_t pool_size,
+    int numa_node_id)
+    : c10::ThreadPool(pool_size, numa_node_id) {}
+
+void PTThreadPool::init_thread() {
+  c10::setThreadName("PTThreadPool");
+  at::init_num_threads();
+}
+
+namespace {
+
+std::shared_ptr<TaskThreadPoolBase> createC10ThreadPool(
```

Comment:
it's a bit sketchy that we ignore some of the parameters here. Maybe put AT_ASSERT that create_new==false?

Reply:
fix this comment?

Reply:
will upd